### ISSUE TITLE
Incorrectly parses Windows file paths

### ISSUE THREAD

**Issue by davidanthoff** [friedemannsommer/lcov-parser#140]
It seems that the parsing at the moment strips away drive letters in paths on Windows. An example coverage file might look like this:
```
SF:C:\workdir\usr\share\julia\stdlib\v1.12\nghttp2_jll\src\nghttp2_jll.jl
LH:0
LF:0
end_of_record
```
And then after parsing the path is given as `\workdir\usr\share\julia\stdlib\v1.12\nghttp2_jll\src\nghttp2_jll.jl`, i.e. it is missing the drive letter `C:`.

vtjnash identified https://github.com/friedemannsommer/lcov-parser/blame/8a980f27039fd41ba6188d37888b1ec9331ec04d/src/parser.ts#L229-L230 as the likely culprit.

**Comment by friedemannsommer** [comment]
@davidanthoff Thanks for pointing that out. This is an oversight on my part.
I will implement a fix for this and let you know when it's published.

**Comment by friedemannsommer** [comment]
@davidanthoff The issue has been fixed in version `4.0.1`, thanks for reporting it.
